sl/check-property.sh.in

```diff
@@ -249,9 +249,11 @@ report_unsafe() {
         "$PRP_FILE" >&2
     fi
 
-    sed -i '' s/__SRCFILEHASH__/"${HASH}"/ "$XMLTRACE"
-    sed -i '' s/__SPECIFICATION__/"${SPEC}"/ "$XMLTRACE"
-    sed -i '' s/__ARCHITECTURE__/"${ARCH}bit"/ "$XMLTRACE"
+    if ! test -z "$XMLTRACE"; then
```

Comment:
Thanks!  I would prefer `test -n ...` over `! test -z ...`.  Can I change it on your behalf?

Reply:
Well, I cannot change it because I do not have write access to this repo.  I messed up Github notifications.  Sorry for the noise.

Reply:
Alright, I'll change it and do a force-push so this stays within a single commit.

Reply:
Thanks for the fixup!